apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -34,6 +34,36 @@ export default function RubricContainer({
   const [teacherHasEnabledAi, setTeacherHasEnabledAi] = useState(
     initialTeacherHasEnabledAi
   );
+  const [aiEvaluations, setAiEvaluations] = useState(null);
+
+  const fetchAiEvaluations = useCallback(() => {
```

Comment:
I did not modify this logic at all and simply copy-pasted it from `RubricContent`